### ISSUE TITLE
react-native-reanimated not getting transpiled

### ISSUE THREAD

**Issue by jcamden** [storybookjs/addon-react-native-web#48]
Hello! After adding a story for a component that uses react-native-reanimated (2.13.0), I'm getting this error:
```
ModuleBuildError: Module build failed (from ../../node_modules/babel-loader/lib/index.js):
SyntaxError: [local paths]/node_modules/react-native-reanimated/lib/index.web.js: Export namespace should be first transformed by `@babel/plugin-proposal-export-namespace-from`.
  4 | export * from './reanimated1';
  5 | export * from './reanimated2';
> 6 | export * as default from './Animated'; // If this line fails, you probably forgot some installation steps. Check out the installation guide here: https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/installation/ 1) Make sure reanimated's babel plugin is installed in your babel.config.js (you should have 'react-native-reanimated/plugin' listed there - also see the above link for details) 2) Make sure you reset build cache after updating the config, run: yarn start --reset-cache
    |        ^^^^^^^^^^^^
  7 |
```
It seems like node_modules/react-native-reanimated/lib/index.web.js is not getting transpiled for some reason. Here are the addons in my main.js:
```
 addons: [
    '@storybook/addon-essentials',
    {
      name: '@storybook/addon-react-native-web',
      options: {
        modulesToTranspile: ['dripsy', '@dripsy', 'react-native-reanimated'],
        babelPlugins: ['react-native-reanimated/plugin'],
        projectRoot: path.resolve(__dirname, '../../../'),
      },
    },
    {
      name: 'storybook-addon-next',
      options: {
        nextConfigPath: path.resolve(
          __dirname,
          '../../../apps/next/next.config.js'
        ),
      },
    },
  ],
```
(I added storybook-addon-next for solito/image. I don't think that would be an issue, but I included it here, just in case.) The relevant parts (I think) of my directory structure are like this:
- apps
- packages
   - app
   - design-system
      - .storybook
      - node_modules
- node_modules

Any chance that it's obvious to you what I might be doing wrong? If not, I'm happy to try to distill this down to a repro repo.

**Comment by jcamden** [comment]
It turns out that all I had to do was install @babel/plugin-proposal-export-namespace-from and add that to the `babelPlugins` array of this addon 🎉 I hadn't tried that because @babel/plugin-proposal-export-namespace-from is included in @babel/preset-env which, in turn, is included in the default Storybook babel config. So, to be honest, I don't know why this worked, or why it wasn't working already; but I'm glad I got it working. Thanks for this awesome addon!
